torchserve/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d5400c22-0f0a-4ce4-894d-c3cda48140e9",
+  "app_id": "torchserve",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor the health and performance of TorchServe",
+    "title": "TorchServe",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
```

Comment:
Windows is supported for this integration right? Not sure why but the Windows CI has not been running.

Reply:
Oh weird, will investigate a bit later if you don't mind!

Reply:
Could it be because `.github/workflows/test-all.yml` has only linux platform?

Reply:
Correct, but we do not really need to run them on windows imo

Reply:
We have the same behaviour with the other tests. Marking as resolved for this PR